ipatests/prci_definitions/nightly_f29.yaml

```diff
@@ -1229,6 +1241,18 @@ jobs:
         timeout: 9000
         topology: *master_2repl_1client
 
+  fedora-29/nfs:
+    requires: [fedora-29/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-29/build_url}'
+        test_suite: test_integration/test_nfs.py::TestNFS
+        template: *ci-master-f29
+        timeout: 9000
+        topology: *master_2repl_1client
+
```

Comment:
This already exists in the file on line https://github.com/freeipa/freeipa/blob/master/ipatests/prci_definitions/nightly_f29.yaml#L1220

Reply:
@rcritten 
thanks for the review. You're right, I somehow didn't catch this one. PR updated.